Code/Tools/FBuild/FBuildCore/Graph/SettingsNode.cpp

```diff
@@ -81,30 +82,38 @@ SettingsNode::~SettingsNode() = default;
     return false;
 }
 
+// GetBrokeragePath
+//------------------------------------------------------------------------------
+const AString & SettingsNode::GetBrokeragePath() const
+{
+    return m_BrokeragePath;
+}
+
 // GetCachePath
 //------------------------------------------------------------------------------
 const AString & SettingsNode::GetCachePath() const
 {
-    // Settings() bff option overrides environment variable
-    if ( m_CachePath.IsEmpty() == false )
+    // Environment variable takes priority
+    if ( m_CachePathFromEnvVar.IsEmpty() == false )
     {
-        return m_CachePath;
+        return m_CachePathFromEnvVar;
     }
-    return m_CachePathFromEnvVar;
+    return m_CachePath;
```

Comment:
The current order of precedence is the documented and expected behavior that many users rely on, so this can't be changed. Any new variables should behave the same way for consistency.

Reply:
This was the result of a bad merge, I think. Oops, sorry!